src/context/WalletProvider/NativeWallet/types.ts

```diff
@@ -22,12 +20,4 @@ export interface LocationState {
   }
 }
 
-export interface NativeSetupProps
-  extends RouteComponentProps<
-    {},
-    any, // history
-    LocationState
-  > {
-  vault: Vault
-  dispatch: React.Dispatch<ActionTypes>
-}
+export type NativeSetupProps = RouteComponentProps<{}, any, LocationState>
```

Comment:
Left was wrong, none of these are passed/consumed, probably leftover from some old implementation